src/utils.js

```diff
@@ -0,0 +1,5 @@
+function randomIndex(min, max) {
```

Comment:
@Bebersohl eslint doesn't like the way you wrote it, so I did this instead:

`exports.randomInt = (min, max) => Math.floor(Math.random() * ((max - min) + 1)) + min;`